lib/Client.js

```diff
@@ -98,6 +98,7 @@ class Client extends EventEmitter {
     * @arg {Object} [options.agent] A HTTP Agent used to proxy requests
     * @arg {Number} [options.maxReconnectAttempts=Infinity] The maximum amount of times that the client is allowed to try to reconnect to Discord.
     * @arg {Function} [options.reconnectDelay] A function which returns how long the bot should wait until reconnecting to Discord.
+    * @arg {Number | String[]} [options.intents] A list of intents, or raw bitmask value describing the intents to subscribe to. "presence" intent must be enabled on your application's page to be used.
```

Comment:
`Guild members` also needs to be enabled on the application's page to be used as per https://github.com/discordapp/discord-api-docs/issues/1363#issue-565525845